conda-forge.yml

```diff
@@ -1,12 +1,13 @@
 build_platform:
   linux_aarch64: linux_64
+  linux_ppc64le: linux_64
```

Comment:
This doesn't work here. We already discovered that in PR ( https://github.com/conda-forge/numcodecs-feedstock/pull/81 ). Reverting...

```suggestion
```